guides/source/configuring.md

```diff
@@ -370,11 +370,9 @@ Sets up the application-wide encoding. Defaults to UTF-8.
 Sets the exceptions application invoked by the `ShowException` middleware when an exception happens.
 Defaults to `ActionDispatch::PublicExceptions.new(Rails.public_path)`.
 
-Exceptions applications need to handle `ActionDispatch::Http::MimeNegotiation::InvalidType` errors, which are raised when a client sends an invalid `Accept` or `Content-Type` header.
-The default `ActionDispatch::PublicExceptions` application does this automatically, setting `Content-Type` to `text/html` and returning a `406 Not Acceptable` status.
-Failure to handle this error will result in a `500 Internal Server Error`.
+Failure to handle given error will result in a `500 Internal Server Error`.
```

Comment:
We can remove all the doc from line 373 to 403 (Sorry I can't highlight with the GitHub UI from this diff). Those were only useful as example on how to properly create an exception app to handle `InvalidType` exception.